### PR TITLE
Ui 0 loader

### PR DESCRIPTION
**Pull request by etolbakov**

### CODE REVIEW

File: 1config-ui/src/com/brunobonacci/oneconfig/ui/controller.cljs [hunk 1]
```diff
@@ -40,13 +40,12 @@
...
     ;; one of :listing, :new-entry-mode, :show-entry-mode
     :client-mode :listing
-                                        ;:new-version-flag? nil
+     ;:new-version-flag? nil
     :1config-version {:current "" :latest ""}
 
     :new-entry empty-new-entry
 
     ;; modal window (initial as plain, should be nested) it should be  ":show-entry-mode"
-    :page-key    :surface-13
```
Comment (etolbakov) [inline 1config-ui/src/com/brunobonacci/oneconfig/ui/controller.cljs]: forgot to remove it

File: 1config-ui/resources/public/css/style.css [hunk 2]
```diff
@@ -85,13 +127,16 @@ body {
...
 }
 
 .ui.divider {
     text-transform: none;
 }
 
+.ui.segment {
+    margin-left: 19px;
+}
+
```
Comment (etolbakov) [inline 1config-ui/resources/public/css/style.css]: this change is required for this indentation 
Screenshot 2019-12-27 at 08 07 14

File: 1config-ui/src/com/brunobonacci/oneconfig/ui/view.cljs [hunk 3]
```diff
@@ -369,4 +371,6 @@
   (ctl/get-all-configs!)
   (ctl/get-version!)
   (reagent/render [app-root]
-                  (. js/document (getElementById "app"))))
+                  (. js/document (getElementById "app")))
+  (ctl/hide-element "entry-mode-menu")
+  (ctl/hide-element "loader"))
```
Comment (etolbakov) [inline 1config-ui/src/com/brunobonacci/oneconfig/ui/view.cljs]: not sure maybe this looks better
```
  (doseq [el ["entry-mode-menu" "loader"]]
    (ctl/hide-element el))
```

Comment (BrunoBonacci) [reply 1config-ui/src/com/brunobonacci/oneconfig/ui/view.cljs]: maybe an over-kill for just 2 elements

**Pull request merged**